DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 `Last update: 2018-07`
 
-# Service owners
+# _Mozilla IAM_ Program Strategy
 
-High level service and product owners for Mozilla IAM.
+* @jeffbryner (IT, InfoSec)
+* @hmitsch (Open Innovation, ParSys)
```

Comment:
would like a single top level capability owner, since this is only used when there is strong disagreement otherwise.
its not possible to solve with more than one decider in this case

note that in theory, we pretty much never get there and i hope we never do :)

Reply:
I understand what you mean. It's the difficult part of writing things down.
If there ever was a Security related issue, Jeff is the undisputed top decider.
On most (all?) other topics Jeff and I put a lot of effort in making sure we show up in _one voice_. I don't know how to handle this. That's why I suggest to have Jeff as Service Owner and call out that Programmatic Strategy is co-shaped.

Reply:
see jeff comments below
in this case, its more than just security, we took this, did the hard work, because iam is intrinsically a security control, not because iam has security topics.